packages/tabs/src/types.ts

```diff
@@ -6,57 +6,57 @@
  */
 
 import { IUseSelectionProps, IUseSelectionReturnValue } from '@zendeskgarden/container-selection';
-import { HTMLProps, ReactNode, RefObject } from 'react';
+import { HTMLProps, MutableRefObject, ReactNode } from 'react';
 
-export interface IUseTabsProps<Item> extends Omit<IUseSelectionProps<Item>, 'direction'> {
+export interface IUseTabsProps<Value> extends Omit<IUseSelectionProps<Value>, 'direction'> {
+  /** Provides an ordered list of unique selection values */
+  values: Value[];
```

Comment:
Shouldn't `useTabs` take a required `tabs` prop instead? It would have a similar shape to `options` on `useCombobox`. Something like:

```
interface ITab {
  value: Value;
  disabled?: boolean;
}

tabs: ITab[];
```

Reply:
I didn't think it was necessary since the eventual `<Tabs />` [child filtering](https://github.com/zendeskgarden/react-components/compare/main...george/tab-selection#diff-e376f924d9ea72b6e21055b3f0ef2e747e758e6c9a4e2c955e8bb7bed71cd19cR27) can exclude disabled tabs from the resulting `values` array. If it does take this shape, I'm foreseeing that we would need to filter out `disabled` tab values before passing them into `useSelection`, since `container-selection` currently doesn't handle `disabled` items at all. It does feel like a container concern, but I'm curious what you think.

Reply:
Should be added now :)